addons/project/models/project_milestone.py

```diff
@@ -47,10 +52,10 @@ def _compute_is_deadline_future(self):
     @api.depends('task_ids.milestone_id')
     def _compute_task_count(self):
         all_and_done_task_count_per_milestone = {
-            milestone.id: (count, is_closed_list.count(True))
+            milestone.id: (count, sum(is_closed_list.count(state) for state in CLOSED_STATES))
```

Comment:
```suggestion
            milestone.id: (count, sum(state in CLOSED_STATES for state in state_list))
```
Should call it `state_list` now.
This way sounds more pythonian, I think :neckbeard: